packages/contracts/scripts/prebuild/export-smtchecker-contracts.ts

```diff
@@ -0,0 +1,46 @@
+import fs from "fs";
+import path from "path";
+
+const foundry_file = path.join(__dirname, "../../foundry.toml");
+
+const getAllFiles = function (dirPath: string, arrayOfFiles: string[]) {
+  const fullPath = path.join(__dirname, dirPath);
+  const files = fs.readdirSync(fullPath);
+
+  arrayOfFiles = arrayOfFiles || [];
+
+  files.forEach(function (file) {
+    if (fs.statSync(fullPath + "/" + file).isDirectory()) {
+      arrayOfFiles = getAllFiles(dirPath + "/" + file, arrayOfFiles);
+    } else {
+      arrayOfFiles.push(path.join(dirPath, "/", file));
+    }
+  });
+
+  return arrayOfFiles;
+};
+
+const main = () => {
+  const buffer = fs.readFileSync(foundry_file);
+  const fileContent = buffer.toString();
+  let contracts = getAllFiles("../../src", []);
```

Comment:
I don't get why we pass that empty array

Reply:
I think we should use [glob](https://www.npmjs.com/package/glob) instead cos here we are also checking Mock and interfaces we could easily solve that just passing a glob like `src/**/!(Mock*|I*)/*.sol`